stdlib/public/runtime/SwiftRT-COFF.cpp

```diff
@@ -36,16 +37,18 @@
   static uintptr_t __stop_##name = 0;
 
 extern "C" {
-DECLARE_SWIFT_SECTION(sw5prt)
```

Comment:
@compnerd @augusto2112 I'm not sure I understand how this part works. I thought the COFF sections had to be declared with abbreviated names?

Reply:
Ugh, unified diffs ... I misread that.  Yes, the section names are different, thats why we need the two files, but everything else should be the same.